parsec/core/gui/tr/parsec_en.po

```diff
@@ -2380,18 +2380,18 @@ msgstr "Reencrypt the workspace"
 msgid "ACTION_WORKSPACE_REENCRYPTION_CONFIRM"
 msgstr "Reencrypt"
 
-msgid "TEXT_WORKPACE_REENCRYPT_OFFLINE_ERROR"
```

Comment:
make a separate PR for this (this one is already big enough ^^)

Reply:
Done :+1: